### DISCUSSION TITLE
Question: Can u scrape Search Playlists

### DISCUSSION THREAD

**Discussion by Tomato6966** [lavalink-devs/Lavalink#606] [Q&A] [answered]
If so how?

**Comment by devoxin** [comment, marked as answer]
Your question needs elaboration. Assuming you're asking whether you can search for tracks, you can, but only `YouTube` and `SoundCloud` support searching. This can be achieved with `ytsearch:QUERY` and `scsearch:QUERY` respectively. Both queries will yield a playlist containing the first search page of tracks.
